gsa/src/web/pages/results/row.js

```diff
@@ -60,6 +56,16 @@ const Row = ({
   const {host} = entity;
   const shown_name = is_defined(entity.name) ? entity.name : entity.nvt.oid;
   const has_tags = is_defined(entity.nvt) && is_defined(entity.nvt.tags);
+  var hostname_elem;
+  if (host.hostname.length > 40)
```

Comment:
You don't have to check the length here. Just render
```jsx
<TableRow>
....
<span title={host.hostname}>
  ({shorten (host.hostname, 40)})
</span>
```
below.
`<span>...</span>` is the same as `<span title={undefined}>....</span>`
and `shorten(undefined)` returns an empty string. Thus your conditions here can be omitted.